### ISSUE TITLE
Provide or update website w/ documentation that is current for what is currently documented.

### ISSUE THREAD

**Issue by takamori** [ReactiveX/rxjs-docs#164]
The website is out of date compared to the checked-in code. Example: Observable.subscribe isn't documented yet, even tho there is some jsdoc in the code if one digs through enough.

Creating a new issue for this since we were asked to do so in https://github.com/ReactiveX/rxjs/issues/2656#issuecomment-345958269

**Comment by ladyleet** [comment]
@takamori thank you for this! we will be discussing this.

**Comment by takamori** [comment]
any updates?

**Comment by valorkin** [comment]
I am working on generating docs from jsdoc and type and translations

**Comment by scottescue** [comment]
Is there a timeline for getting http://reactivex.io/rxjs updated? Is this anything a RxJS noob like myself could help with to move this along?

I can tell you as a new RxJS user it's extremely confusing, and ultimately frustrating, when the published documentation doesn't include core methods that are referenced in numerous tutorials and code samples.

**Comment by ladyleet** [comment]
@scottescue yep we are working on it! you can go ahead and check out the issues and see if there is anything you can contribute too! happy to help
